README.md

```diff
@@ -5,33 +5,23 @@ ffagc
 
 ## Initial Environment Setup
 
-build / install stuff I dunno who even uses rails any more:
+Set up the project with:
 
-    rake
-    bundle update
-    bundle install
+```sh
+  bundle install
+  bundle exec rake db:migrate RAILS_ENV=development
+  bundle exec rails server
+```
 
-You'll need to create config/secrets.yml with the form:
-
-    development:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    test:
-      secret_key_base: [[[LOTS OF HEX]]]
-
-    production:
-      secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-
-Use 'rake secret' to generate the hex.  Don't check this file into source control!
+If you are running this project in production you'll need to set environment variables for secrets that include `ENV` in `config/secrets.yml`.
 
 ## Email Setup
 
-If you plan to send email, you'll need to create config/initializers/smtp_secret.rb
-with:
-
-    ENV['PASSWORD']="your smtp password"
+If using SMTP set the required ENV variables or fill in (and do not commit) `.env` like so:
 
-and edit config/environments/*.rb to include your smtp server information.
```

Comment:
This line is obsolete. If you need to override any secrets or configuration values look at `config/secrets.yml`. There are now a bunch of lines in there that look like `password: <%= ENV['SMTP_PASSWORD'] || 'this_is_not_the_password' %>`. To override the SMTP password you would set SMTP_PASSWORD in your ENV. Often in production environments that would be done with some specific infrastructure for secrets. In your case you probably just want to make a `.env` file (which is automatically read by the `dotenv` gem) that includes lines such as `SMTP_PASSWORD='this_is_not_the_password'`. I did cover this in updated Readme around this line https://github.com/ywwg/ffagc/pull/61/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20